src/main/Config.cpp

```diff
@@ -276,14 +276,8 @@ readArray(ConfigItem const& item)
 
 template <typename T>
 T
-readInt(ConfigItem const& item, T min = std::numeric_limits<T>::min(),
-        T max = std::numeric_limits<T>::max())
+castInt(int64_t v, ConfigItem const& item, T min, T max)
```

Comment:
could just pass string instead of ConfigItem

Reply:
Makes sense. Updated it.